modules/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -103,6 +103,8 @@ public function loop_session_events() {
 								'pq': '" . esc_js( $data_instance['quantity'] ) . "',
 								'pt': '" . esc_js( $product_details['type'] ) . "',
 								'ui': '" . esc_js( $this->get_user_id() ) . "',
+								'url': '" . esc_js( home_url() ) . "',
+								'woo_version': '" . esc_js( WC()->version ) . "',
```

Comment:
Before we use it here and below, maybe we should check and ensure WC version is a valid object, like we did in #10538?

Reply:
I don't think that's necessary - in #10538 the check is on `WC()->session` before we call methods on `session`. In this case we are reading the value of `WC()->version` (e.g. string or worst case null/undefined) as a property. 

Note this analytics code is all [conditional on Woo being active](https://github.com/Automattic/jetpack/blob/master/modules/woocommerce-analytics/wp-woocommerce-analytics.php#L62), so `WC()` should be fine.